models/discriminators.py

```diff
@@ -67,36 +67,40 @@ def __init__(self, img_size, kernel_size, num_filters, lr):
                 lr : float
         """
         super().__init__()
+
+        self.w_init = tf.random_normal_initializer(stddev=0.02)
+        self.bn_init = tf.random_normal_initializer(1., 0.02)
+
         self.img_size = img_size
         num_channels = self.img_size[0]
         # self.xent_loss_fn = tf.keras.losses.BinaryCrossentropy(from_logits=True)
-        self.optimiser = tf.keras.optimizers.Adam(lr)
+        self.optimiser = tf.keras.optimizers.Adam(lr, beta_1=0.5)
```

Comment:
Is `0.5` the number from the paper? The standard Adam `beta_1` is 0.9

Reply:
Yup, taken from the paper. Just wanted to get it training properly so replicated as much as possible.